utils/ort/src/funTest/kotlin/storage/S3FileStorageFunTest.kt

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2019 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
S3FileStorageFunTest.kt is a new file so please change copyright year from 2019 to 2023

Reply:
Addressed by squashing